src/avif.c

```diff
@@ -560,17 +560,17 @@ avifBool avifCropRectConvertCleanApertureBox(avifCropRect * cropRect,
     clapFraction croppedCenterY = clapFractionAdd(uncroppedCenterY, vertOff);
 
     clapFraction halfW;
-    halfW.n = widthN;
-    halfW.d = widthD * 2;
```

Comment:
If the original code was written this way to avoid divisions, I can revert these two changes.

Reply:
Yes, my intent here was to avoid division. All of the other changes are good though.

Reply:
Thanks for the reply. I will remove these two changes from this PR. Note that a good compiler should be able to combine these divisions with the earlier modulo operations when generating code for x86:

```
    if ((widthN % widthD) != 0) {
        avifDiagnosticsPrintf(diag, "[Strict] clap width %d/%d is not an integer", widthN, widthD);
        return AVIF_FALSE;
    }
    if ((heightN % heightD) != 0) {
        avifDiagnosticsPrintf(diag, "[Strict] clap height %d/%d is not an integer", heightN, heightD);
        return AVIF_FALSE;
    }
```